src/lib.rs

```diff
@@ -771,6 +771,23 @@ impl Account {
         Ok(())
     }
 
+    /// Yield the profiles supported according to the account's server directory
+    ///
+    /// The first element of the tuple is the profile name, the second is a URL pointing to a
+    /// human-readable description of the profile.
```

Comment:
I think we can't promise that the 2nd tuple value is a URL, the draft spec says:

> The contents of these human-readable descriptions are up to the CA; for example, they might be prose descriptions of the properties of the profile, or the might be URLs pointing at a documentation site. ACME Clients SHOULD present these profile names and descriptions to their operator during initial setup and at appropriate times thereafter.